src/models/DefaultFilters.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const ElectronConfig = require('electron-config')
+const configName = process.env.NODE_ENV === 'test' ? 'config-test' : 'config'
+const storage = new ElectronConfig({ name: configName })
+const Filter = require('./Filter')
+const LastFilter = require('./LastFilter')
+const DEFAULT_FILTER_KEY = 'default-filters'
+
+class DefaultFilters {
+  static init(login) {
+    if (!storage.has(DEFAULT_FILTER_KEY)) {
+      this.myIssuesFilter(login)
+      this.myCommentsFilter(login)
+      this.myAssignmentsFilter(login)
+      this.myMentionsFilter(login)
+      this.myPopularFilter(login)
+      this.save()
+    }
+    return
+  }
+
+  static myIssuesFilter(login) {
+    const defaultFilter = new Filter('My Issues')
+    defaultFilter.store(`author:${login} is:open sort:updated-desc`)
+    LastFilter.save(defaultFilter.key)
+  }
+
+  static myCommentsFilter(login) {
+    const defaultFilter = new Filter('My Comments')
+    defaultFilter.store(`commenter:${login} sort:updated-desc`)
```

Comment:
Are you doing the `filterNames.forEach` loop inside the init method that has the `login` parameter?

Could try converting DefaultFilters into a class with a `constructor(login)`, set `this.login = login` in it, and make the method doing the `filterNames.forEach` loop a regular instance method (drop the `static`). Then you could use `this.login` in it, and the `this` even inside the `forEach` block, since you're doing the new-style `=> {` function syntax, should refer to the instance of the class and not the `forEach` function.